<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="NAVEENKUMARAN" failures="1" tests="4" name="com.hotelbooking.tests.BookingTest" time="3.114" errors="0" timestamp="2024-12-31T20:20:53 IST" skipped="2">
  <testcase classname="com.hotelbooking.tests.BookingTest" name="validateAssertions" time="0.367">
    <failure message="The following asserts failed:
	BOOKINGDATES should be {&quot;checkin&quot;:&quot;2024-02-23&quot;,&quot;checkout&quot;:&quot;2024-10-23&quot;}, but it&#039;s not expected [{&quot;checkin&quot;:&quot;2024-02-23&quot;,&quot;checkout&quot;:&quot;2024-10-23&quot;}] but found [null]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
BOOKINGDATES should be {"checkin":"2024-02-23","checkout":"2024-10-23"}, but it's not expected [{"checkin":"2024-02-23","checkout":"2024-10-23"}] but found [null]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at base.BaseTest.validateAssertions(BaseTest.java:143)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- validateAssertions -->
  <system-out/>
  <testcase classname="com.hotelbooking.tests.BookingTest" name="createBookingId" time="2.747"/>
  <system-out/>
  <testcase classname="com.hotelbooking.tests.BookingTest" name="partialUpdateBookingId">
    <skipped/>
  </testcase> <!-- partialUpdateBookingId -->
  <system-out/>
  <testcase classname="com.hotelbooking.tests.BookingTest" name="updateBookingId">
    <skipped/>
  </testcase> <!-- updateBookingId -->
  <system-out/>
</testsuite> <!-- com.hotelbooking.tests.BookingTest -->
